x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/planner/ExpressionTranslators.java

```diff
@@ -457,24 +455,8 @@ protected Query asQuery(ScalarFunction f, TranslatorHandler handler) {
         }
 
         public static Query doTranslate(ScalarFunction f, TranslatorHandler handler) {
```

Comment:
Should we mark this deprecated (for removal) while we're at it, or outright remove it? This does nothing useful now.